FordLabs/peoplemover
# Move currentUser state from redux to recoil

## Issue
Resolves N/A 

## What was done
- [x] Move currentUser state from redux to recoil

## How to test
1. Ensure that the app works as it did before.
2. Ensure the users name shows up on the right hand side of the header when logged in.
<img width="347" alt="Screen Shot 2022-05-20 at 11 11 47 AM" src="https://user-images.githubusercontent.com/17144844/169558365-0279277c-47dc-4681-8e12-a4b843b8904f.png">
